libs/shared/ui/src/lib/components/inputs/input-select/input-select.tsx

```diff
@@ -156,7 +156,7 @@ export function InputSelect({
   }
 
   const MultiValue = (props: MultiValueProps<Value, true, GroupBase<Value>>) => (
-    <span className="text-sm text-neutral-400 mr-1">
+    <span className="flex text-sm text-neutral-400 mr-1">
```

Comment:
Fix UI bug for annotations

<img width="783" alt="Screenshot 2024-05-21 at 09 21 38" src="https://github.com/Qovery/console/assets/533928/5f9dbd27-8926-4f71-95c7-3f1a9d00b477">